gcp_variant_transforms/libs/hashing_util.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generates hashing code for string."""
+
+import sys
+
+import farmhash
+
+
+def generate_int64_hash_code(string):
```

Comment:
I am thinking to make this function a bit more specialized:
`def generate_int64_hash_code(string, min_hash_value = 0, max_hash_value = sys.max_int):`

so that we can call it like `def generate_int64_hash_code('test', 0, 256)` which means I am expecting the hash values to be in the [0, 256) range.

Let's discuss this offline, meanwhile I found this useful:

http://www.cim.mcgill.ca/~langer/250/27-hashing.pdf